extension/snippets/dvc-yaml.code-snippets

```diff
@@ -38,6 +38,19 @@
     ],
     "description": "Foreach stage template for dvc.yaml pipeline"
   },
+  "Matrix Pipeline Stage Template": {
+    "prefix": "dvc-pipeline-matrix-stage",
+    "body": [
+      "  ${1:train}:",
+      "    matrix:",
+      "      model: [${2:cnn}, ${3:xgb}]",
+      "      feature: [${4:feature1}, ${5:feature2}, ${6:feature3}]",
+      "    cmd: ${7:./train.py} --feature \\${item.feature}} \\${item.model}",
+      "    outs:",
+      "      - \\${item.model}.pkl"
```

Comment:
Got the example from the docs: https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#matrix-stages

Reply:
I would also put the cursor here (if possible) so the user can change the outs.